northstar/src/runtime/fork/forker/messages.rs

```diff
@@ -25,12 +25,18 @@ pub enum Message {
         path: NonNullString,
         args: Vec<NonNullString>,
         env: Vec<NonNullString>,
-        stdout: PathBuf,
-        stderr: PathBuf,
+        output: ProcessOutput,
     },
     ExecResult, // TODO
 }
 
+/// Process output
+#[derive(Debug, Serialize, Deserialize)]
+pub enum ProcessOutput {
+    Files { stdout: PathBuf, stderr: PathBuf },
+    Pty { slave: PathBuf },
```

Comment:
This is good place to document the two flavours and when each one is used.

Reply:
closing discussion because this got replaced by the 3 fd array